### PR TITLE
Add install addon string

### PR DESCRIPTION
**Pull request by tymmesyde**

### CODE REVIEW

File: fi-FI.json
Comment (TRtomasz) [inline fi-FI.json:1]: what has happend here

Comment (tymmesyde) [reply fi-FI.json:1]: spacings, someone published the file with new lines just like en-US.json
`add-all.sh` removed them

**Review by jaruba — APPROVED** [review]

**Comment by jaruba** [comment]
@tymmesyde there are conflicts in the chinese translations

**Comment by jaruba** [comment]
this string was added with this PR: https://github.com/Stremio/stremio-translations/pull/714

**Pull request closed without merging**